easybuild/easyconfigs/s/SpectrA/SpectrA-0.9.0-GCC-10.2.0.eb

```diff
@@ -0,0 +1,34 @@
+# EasyBuild easyconfig
+#
+# Fred Hutchinson Cancer Research Center - Seattle - Washington - US
+# https://www.fredhutch.org
+# John Dey <jfdey@fredhutch.org>
+#
+easyblock = 'CMakeMake'
+
+name = 'SpectrA'
+version = '0.9.0'
+
+homepage = 'https://spectralib.org/'
+description = """Spectra stands for Sparse Eigenvalue Computation Toolkit as a Redesigned ARPACK. It is a C++
+ library for large scale eigenvalue problems, built on top of Eigen, an open source linear algebra library."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
```

Comment:
@fizwit `Eigen 3.3.9` is only available for `10.3.0` currently.

Reply:
what do you suggest?  Can I submit Eigen 3.3.9 for 10.2 or do I need to move this to 10.3.0 ?